brushfire-core/src/main/scala/com/stripe/brushfire/Tree.scala

```diff
@@ -1,14 +1,14 @@
 package com.stripe.brushfire
 
+import com.stripe.bonsai.FullBinaryTreeOps
 import com.twitter.algebird._
 
-// type Tree[K, V, T] = AnnotatedTree[K, V, T, Unit]
```

Comment:
We definitely use it in our other projects, but maybe not within Brushfire itself.